vignettes/extras/BugAndPathwayAbundanceAllGutSamples_ar12.19.16.Rmd

```diff
@@ -206,7 +206,7 @@ bardat %<>% mutate(Sample = as.numeric(factor(Sample, levels=factor(firmicutes_o
 set.seed(14)
 b <- ggplot(arrange(bardat, desc(OTU), Abundance), aes(x=Sample, y=Abundance, fill=OTU))
 b <- b + geom_area()
-b <- b + scale_fill_manual(values=sample(pallet, size = 8, replace=F))
```

Comment:
"F" isn't a primitive so could in theory be assigned to something else, even F <- TRUE, so it isn't recommended to use this abbreviation.